README.md

```diff
@@ -565,3 +567,33 @@ module.exports = {
     'eslint --max-warnings=0 ' + files.filter((file) => !cli.isPathIgnored(file)).join(' ')
 }
 ```
+
+#### ESlint >= 7
+
+In versions of ESlint > 7, isPathIgnored is an async function and now returns a promise, see: https://eslint.org/docs/developer-guide/nodejs-api#-eslintispathignoredfilepath. The code below can be used to reinstate the above functionality.
+
+This particular code uses a tiny package, `node-filter-async` (https://www.npmjs.com/package/node-filter-async), to filter the files array with an async function, If you prefer to not have an extra dependency, it is quite simple to write a similar function.
+
+Since 10.5.3 (https://github.com/okonet/lint-staged/releases), any errors due to a bad eslint config will come through to the console.
```

Comment:
Can we hide the link behind the version number?

```suggestion
Since [v10.5.3](https://github.com/okonet/lint-staged/releases/tag/v10.5.3) any errors due to a bad eslint config will come through to the console.
```